app.js

```diff
@@ -0,0 +1,58 @@
+// var ip = document.querySelector("#ip");
+// var op = document.querySelector("#op");
```

Comment:
Remove unnecessary code, though it is commented but it is not a good practice to follow

Reply:
Issue resolved.